features/preferences/impl/src/main/res/values/localazy.xml

```diff
@@ -13,7 +13,7 @@
   <string name="screen_advanced_settings_send_read_receipts">"Read receipts"</string>
   <string name="screen_advanced_settings_send_read_receipts_description">"If turned off, your read receipts won\'t be sent to anyone. You will still receive read receipts from other users."</string>
   <string name="screen_advanced_settings_share_presence">"Share presence"</string>
-  <string name="screen_advanced_settings_share_presence_description">"If turned off, you won’t be able to send or receive read receipts or typing notifications"</string>
+  <string name="screen_advanced_settings_share_presence_description">"If turned off, you won’t be able to send or receive read receipts or typing notifications."</string>
```

Comment:
This is a bit weird, the other option in this screen has an ending `.` at the end of its description.